python/ray/autoscaler/kubernetes/example_scripts/run_local_example.py

```diff
@@ -0,0 +1,56 @@
+from collections import Counter
+import sys
+import time
+import ray
+""" Run this script locally to execute a Ray program on your Ray cluster on
+Kubernetes.
+
+Before running this script, you must port-forward from the local host to
+the relevant Kubernetes head service e.g.
+kubectl -n ray port-forward service/example-cluster-ray-head 10001:10001.
+
+Set the constant LOCAL_PORT below to the local port being forwarded.
+"""
+LOCAL_PORT = 10001
+
+
+@ray.remote
+def gethostname(x):
+    import platform
+    import time
+    time.sleep(0.01)
+    return x + (platform.node(), )
+
+
+def wait_for_nodes(expected):
+    # Wait for all nodes to join the cluster.
+    while True:
+        num_nodes = len(ray.nodes())
```

Comment:
I think we are trying to move away from `ray.nodes()`? (`len(ray.nodes())` also counts dead nodes.). Do you think we can use `ray.cluster_resources()`?

Reply:
Will do  (copy pasted example code from many months) ago.

Reply:
yeah, it's kinda bad, but we definitely have lots of buggy examples/code that use `ray.nodes()` incorrectly.